dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -80,3 +80,12 @@ preferCRLs.desc=By default, OCSP is the preferred mechanism for checking revocat
 
 disableFallback=No fallback
 disableFallback.desc=Disable the certificate revocation status checker fallback mechanism. During certificate revocation status check, the default behavior is fallback and check by using the non-preferred mechanism (OCSP or CRLs). Default is false.
+
+certCheckerSchedule=Certificate checker schedule
+certCheckerSchedule.desc=Time period for the ACME CA service to check if the certificate is expiring or revoked. If expiring or revoked, the certificate is renewed. If the first renew request does not work, the certificate renewal request continues until a new certificate is received using the timing set by certCheckerErrorSchedule.  Setting the certCheckerSchedule property to zero or a negative value disables automatic certificate renewal.
```

Comment:
**change**
If the first renew request does not work, the certificate renewal request continues until a new certificate is received using the timing set by certCheckerErrorSchedule.
**to**
If the first renew request does not work, the certificate renewal request continues according to the timing set by the certCheckerErrorSchedule attribute until a new certificate is received.

Reply:
Done